__tests__/app.test.js

```diff
@@ -39,14 +53,42 @@ describe('NC News Server', () => {
                     expect(typeof topic.slug).toBe('string')
                 })
             })
-           
         })
-        test('GET 404 : returns error when given wrong path', () => {
+    })
+    describe('GET /api/articles', () => {
+        test('GET 200 : /api/articles returns all articles', () => {
             return request(app)
-            .get ('/api/topic')
-            .expect(404)
-            .then (({ body }) => {
-                expect(body.message).toBe('Path Not Found')
+            .get('/api/articles')
+            .expect(200)
+            .then(({ body }) => {
+                expect(body.articles.length).toBe(13)
+                body.articles.forEach((article) => {
+                    expect(typeof article.author).toBe('string')
+                    expect(typeof article.title).toBe('string')
+                    expect(typeof article.article_id).toBe('number')
+                    expect(typeof article.body).toBe('string')
+                    expect(typeof article.topic).toBe('string')
+                    expect(typeof article.created_at).toBe('string')
+                    expect(typeof article.votes).toBe('number')
+                    expect(article.article_img_url).toMatch(/^https?:\/\/\S+$/)
```

Comment:
Need to make sure these are sorted by date and descending - jest sorted can be used to test this.